app/models/code_snippets.py

```diff
@@ -40,6 +40,94 @@ def get_group_name(model_name, source):
+    "CleTimer": "haimasree/kipoi-docker:sharedpy3keras2",
+    "SiSp": "haimasree/kipoi-docker:sharedpy3keras2",
+    "FactorNet": "haimasree/kipoi-docker:sharedpy3keras2",
+    "pwm_HOCOMOCO": "haimasree/kipoi-docker:sharedpy3keras2",
+    "MaxEntScan": "haimasree/kipoi-docker:sharedpy3keras2",
+    "DeepBind": "haimasree/kipoi-docker:sharedpy3keras2",
+    "lsgkm-SVM": "haimasree/kipoi-docker:sharedpy3keras2",
+    "rbp_eclip": "haimasree/kipoi-docker:sharedpy3keras2",
+    "MPRA-DragoNN": "haimasree/kipoi-docker:mpra-dragonn",
+    "extended_coda": "haimasree/kipoi-docker:extended_coda",
+    "MMSplice/pathogenicity": "haimasree/kipoi-docker:mmsplice",
+    "MMSplice/splicingEfficiency": "haimasree/kipoi-docker:mmsplice",
+    "MMSplice/deltaLogitPSI": "haimasree/kipoi-docker:mmsplice",
+    "MMSplice/modularPredictions": "haimasree/kipoi-docker:mmsplice",
+    "MMSplice/mtsplice": "haimasree/kipoi-docker:mmsplice-mtsplice",
+    "DeepMEL": "haimasree/kipoi-docker:deepmel",
+    "Framepool": "haimasree/kipoi-docker:framepool",
+    "KipoiSplice": "haimasree/kipoi-docker:kipoisplice",
+    "deepTarget": "haimasree/kipoi-docker:deeptarget",
+    "AttentiveChrome": "haimasree/kipoi-docker:attentivechrome",
+    "BPNet-OSKN": "haimasree/kipoi-docker:bpnet-oskn",
+    "SeqVec": "haimasree/kipoi-docker:seqvec",
+    "Xpresso": "haimasree/kipoi-docker:sharedpy3keras2"
+    }
+
+    try:
+        kw = get_example_kwargs(model_name, source)
+    except Exception:
+        kw = "Error"
+    if isinstance(kw, dict):
+        for key, value in kw.items():
+            if isinstance(value, str):
+                kw[key] = value.replace('example', '/app/example')
+    ctx = {"model_name": model_name,
+           "example_kwargs": kw,
+           "batch_size": get_batch_size(model_name, source),
+           "source": source,
+           "model_name_no_slash": model_name.replace("/", "|"),
+           "output_dir" : "example"
+           }
+    try:
+        if model_name in model_group_to_image_dict: # Special provision for MMSplice
+            docker_image_name =  model_group_to_image_dict[model_name]
+        else:
+            docker_image_name = model_group_to_image_dict[model_name.split('/')[0]]
+    except Exception:
+        docker_image_name = ""
+    ctx["docker_image_name"] = docker_image_name
+    print("docker image name {}".format(docker_image_name))
+    return [("Get the docker image", """docker pull {docker_image_name}""".format(**ctx)),
+            ("Get the activated conda environment inside the container",
+             """docker run -it {docker_image_name}""".format(**ctx)
+             ),
+        ("Test the model", "docker run {docker_image_name} kipoi test {model_name} --source={source}".format(**ctx)),
+        ("Make prediction for custom files directly", """mkdir -p <absolute path to your data dir>/output \\
+docker run -v <absolute path to your data dir>:/app/ {docker_image_name} \\
```

Comment:
Replace all instances `<absolute path to your data dir>` with a concrete example directory, say `PWD/kipoi-example`. That way, the user can just run this command and then adjust the directory later. Also add a note that this path can be adjusted to an arbitrary path. Here is one eample: 

```
# Create an example directory containing the data.
mkdir -p $PWD/kipoi-example
# You can replace $PWD/kipoi-example with a different absolute path containing the data.
docker run $PWD/kipoi-example:/app/ ...
```

Reply:
Great idea! Done.